sdks/python/apache_beam/transforms/trigger.py

```diff
@@ -378,6 +378,8 @@ class AfterCount(TriggerFn):
   COUNT_TAG = _CombiningValueStateTag('count', combiners.CountCombineFn())
 
   def __init__(self, count):
+    if not isinstance(count, int) or count < 1:
```

Comment:
<!--thread_id:cc_158342029_t; commit:2df1e910f7e7508a6253afeb297a7487f5d5c392; resolved:0-->
<!--section:context-quote-->
> **aaltay** wrote:
> `isinstance(count, int)` is over limiting, For example both `isinstance(3l, int)` and `isinstance(3.0, int)` would result in `False`. Even though there is no reason to accept them as valid arguments. I do not not think we need to type check things. Checking for the `count < 1` should be enough here.

<!--section:body-->
I am not sure if you mean count = "thirty one"  or count = "three followed by letter L"
The first is rightfully True and the second is rightfully False, so I don't see the problem with that statement.
count < 1 is not restrictive enough, a user could enter 3.4 and they would only get an obscure message thrown by the protobuf. With the checking above, they get a clear reason of their error.

Reply:
I meant a long value as in `type(3L) = <type 'long'>`. I do not see any reason for not letting users to pass long values here.